charts/spinnaker/templates/configmap/halyard-config.yaml

```diff
@@ -132,6 +132,10 @@ data:
                 {{ if $account.readPermissions }}--read-permissions {{ join " --read-permissions " $account.readPermissions }}{{ end }} \
                 {{ if $account.writePermissions }}--write-permissions {{ join " --write-permissions " $account.writePermissions }}{{ end }} \
                 --provider-version v2
+    {{- range $index, $customResource := .Values.kubeConfig.customResources }}
+    $HAL_COMMAND config provider kubernetes account $PROVIDER_COMMAND {{ $name }} --docker-registries dockerhub \
+                --context {{ $account.context }} --provider-version v2 --add-custom-resource={{ $customResource }} --spinnaker-kind=serverGroupManager --no-validate
```

Comment:
Why do we here need `--spinnaker-kind=serverGroupManager`? If not set, what would it default to?

Reply:
Would be nice to make `spinnaker-kind` actually configurable. Then when we add other custom resources we can decide how to map it via configuration.